bundles/com.espressif.idf.debug.gdbjtag.openocd/src/com/espressif/idf/debug/gdbjtag/openocd/dsf/process/monitors/CustomOutputStreamMonitor.java

```diff
@@ -27,9 +27,9 @@ public CustomOutputStreamMonitor(InputStream stream, Charset charset)
 	}
 	
 	@Override
-	protected void startMonitoring()
+	protected void startMonitoring(String processLabel)
 	{
-		super.startMonitoring();
+		super.startMonitoring(processLabel);
 	}
```

Comment:
Does this have backward compatibility with Eclipse 2022-09 and Eclipse 2022-12?

Reply:
It doesn't have backward compatibility